.travis.yml

```diff
@@ -14,6 +14,7 @@ script:
       --tag "sonar-scanner-cli:$SCANNER_VERSION"
       --tag "sonarsource/sonar-scanner-cli:$SCANNER_VERSION"
       --tag "sonarsource/sonar-scanner-cli:latest"
+  - git clone https://github.com/SonarSource/sonar-scanning-examples.git target_repository
```

Comment:
Travis will be removed in the next PR. I just wanted this PR to be checked against already existing CI, because this PR will only be effective when merged (a little bit of a chicken and egg problem).

Reply:
yeah, your comments in the standup made complete sense.